curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
We should also add a <code>quietly()</code> method or something to swallow <code>KeeperException.NoWatcher</code> exceptions. I'll bet most users don't care if the watcher doesn't exist.